lib/index.js

```diff
@@ -73,7 +73,6 @@ const internals = {
 exports.wrap = function (error, statusCode, message) {
 
     Hoek.assert(error instanceof Error, 'Cannot wrap non-Error object');
-    Hoek.assert(!error.isBoom || (!statusCode && !message), 'Cannot provide statusCode or message with boom error');
```

Comment:
Do we want to do something if they pass in a boom error and status or message? If `wrap` doesn't change the existing boom error, it would be a little surprising if two of the 3 passing in arguments had no impact on what the function did.